docs/playbooks/oneclick-aibroker.md

```diff
@@ -0,0 +1,267 @@
+# Install AI Broker Application
+
+## Prerequisites
+
+You will need a RedHat OpenShift v4.14 or above with IBM Maximo Application Suite Core v9.x already be installed, the [oneclick-core](oneclick-core.md) playbook can be used to set this up.
+
+### Dependencies:
+
+* IBM Maximo Application Suite Core v9.x
+
+## Overview
+
+This playbook will add **AI Broker v1.0.x** to an existing IBM Maximo Application Suite Core installation or as standalone instance.
```

Comment:
This says "as a standalone instance" but the Prerequisites above say it should have MAS Core already installed?

Reply:
it is fixed now